framework-docs/src/docs/asciidoc/web-reactive.adoc

```diff
@@ -6,7 +6,7 @@ include::page-layout.adoc[]
 This part of the documentation covers support for reactive-stack web applications built
 on a https://www.reactive-streams.org/[Reactive Streams] API to run on non-blocking
 servers, such as Netty, Undertow, and Servlet containers. Individual chapters cover
-the <<webflux.adoc#webflux, Spring WebFlux>> framework,
+the <<web/webflux.adoc#webflux, Spring WebFlux>> framework,
```

Comment:
```suggestion
the <<webflux, Spring WebFlux>> framework,
```

Your fix works; however, when cross referencing a section within the current file (after includes have been applied), there is no need to use the folder and file name.

I'll simplify that when merging the PR.

Reply:
Thanks for your review.